pkg/apis/camel/v1/kamelet_types.go

```diff
@@ -76,6 +78,15 @@ type Kamelet struct {
 
 // KameletSpec specifies the configuration required to execute a Kamelet.
 type KameletSpec struct {
+	KameletSpecBase `json:",inline"`
+	// the optional versions available for this Kamelet. This field may not be taken in account by Camel core and is meant to support
```

Comment:
cc @claudio4j I've added a more clear documentation to explain this won't be supported ootb by the core. Thanks.

Reply:
Looks good then, thanks.